### PR TITLE
Add upgrade clarification

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/upgrade-index.adoc [hunk 1]
```diff
@@ -16,6 +16,9 @@ Upgrades to cluster-level and node configuration components are performed using
 
 The latest versions of each downloadable package are found in the *Downloads* page, located in Anypoint Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
 
+[IMPORTANT]
+You must be running Runtime Fabric version 1.4.54 in order to upgrade to Runtime Fabric version 1.5. If you are using an earlier version than Runtime Fabric version 1.4.54, you must upgrade to Runtime Fabric version 1.4.54 before upgrading to version 1.5.
```
Comment (yianL) [inline modules/ROOT/pages/upgrade-index.adoc]: "1.5 and above" ?

**Review by yianL — APPROVED** [review]

**Comment by yianL** [comment]
Also, should we be updating the 1.4 docs branch? or 1.5 branch?

File: modules/ROOT/pages/upgrade-index.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/upgrade-index.adoc]: Got it - thank you!

**Comment by paigehargrave** [comment]
@yianL - Good question - I've added it to both versions in case they land on the 1.5 docs and are running 1.4.

File: modules/ROOT/pages/upgrade-index.adoc [hunk 2]
```diff
@@ -16,6 +16,9 @@ Upgrades to cluster-level and node configuration components are performed using
 
 The latest versions of each downloadable package are found in the *Downloads* page, located in Anypoint Runtime Manager in the *Runtime Fabrics* tab. A link to each package’s release notes is provided to show the changes.
 
+[IMPORTANT]
+You must be running Runtime Fabric version 1.4.54 in order to upgrade to Runtime Fabric version 1.5 or later. If you are using an earlier version than Runtime Fabric version 1.4.54, you must upgrade to Runtime Fabric version 1.4.54 before upgrading to version 1.5 or later.
```
Comment (kbostrom) [inline modules/ROOT/pages/upgrade-index.adoc]:
```suggestion
You must be running Runtime Fabric version 1.4.54 to upgrade to version 1.5 or later. If you are using a version earlier than 1.4.54, you must upgrade to 1.4.54 before upgrading to 1.5 or later.
```
Don't think we need to keep repeating "version" and the product name in this short context. Also think we could run with this:
```suggestion
To upgrade to Runtime Fabric version 1.5 or later, you must either already be running version 1.4.54 or first upgrade to 1.4.54 before upgrading to 1.5.
```

**Review by kbostrom — APPROVED** [review]
Here you go!

**Pull request merged**